types/index.d.ts

```diff
@@ -751,6 +751,11 @@ export var CompressionCodecs: {
 }
 
 export class KafkaJSError extends Error {
+  readonly message: Error["message"];
+  readonly name: string;
+  readonly retriable: boolean;
+  readonly helpUrl: string;
```

Comment:
`retriable` and `helpUrl` may not be set, so they should be nullable, right?

```suggestion
  readonly retriable?: boolean;
  readonly helpUrl?: string;
```

Reply:
`retriable` looks to be set as true by default: https://github.com/tulios/kafkajs/blob/master/src/errors.js#L2